nvm.sh

```diff
@@ -1828,6 +1828,17 @@ nvm() {
         NVM_PREFIX="$(nvm_node_prefix)"
         NVM_SUCCESS_MSG="Uninstalled node $VERSION"
       fi
+
+      # Check version dir permissions
+      local INVALID_FILES="$(find "$VERSION_PATH" -exec [ ! -w "{}" ] \; -exec echo "{}" \;)"
+      if [ "$INVALID_FILES" ]; then
```

Comment:
Yeah it works in ksh:

```
% ksh
$ var=""
$ [ $var ] && echo yep || echo nope
nope
```

I also just checked and `-z` won't actually work, it returns true on an empty string which isn't what I want.
```
bash-3.2$ var=""
bash-3.2$ [ $var ] && echo yep || echo nope
nope
bash-3.2$ [ -z $var ] && echo yep || echo nope
yep
```
Personally I reckon this is ok but open to suggestions if there's a better way.